test/foundry/deploy/15_holograph_registry_tests.t.sol

```diff
@@ -0,0 +1,656 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.13;
+
+import {Test, Vm, console} from "forge-std/Test.sol";
+import {Constants} from "../utils/Constants.sol";
+import {RandomAddress} from "../utils/Utils.sol";
+import {Holograph} from "../../../src/Holograph.sol";
+import {Holographer} from "../../../src/enforcer/Holographer.sol";
+import {HolographRegistry} from "../../../src/HolographRegistry.sol";
+import {MockExternalCall} from "../../../src/mock/MockExternalCall.sol";
+import {HolographERC721} from "../../../src/enforcer/HolographERC721.sol";
+import {HolographERC20} from "../../../src/enforcer/HolographERC20.sol";
+import {DeploymentConfig} from "../../../src/struct/DeploymentConfig.sol";
+
+/**
+ * @title Testing the Holograph Registry
+ * @notice Suite of unit tests for the Holograph Registry contract 
+ * @dev Translation of a suite of Hardhat tests found in test/15_holograph_registry_test.ts
+ */
+
+contract HolographRegistryTests is Test {
+    uint256 localHostFork;  
+    string LOCALHOST_RPC_URL = vm.envString("LOCALHOST_RPC_URL");
+    Holograph holograph;
+    HolographRegistry holographRegistry;
+    HolographRegistry holographRegistryNew;
+    HolographRegistry registryDeployedByScript;
+    MockExternalCall mockExternalCall;
+    HolographERC721 holographERC721;
+    HolographERC20 holographERC20;
+    Holographer sampleErc721Holographer;
+
+    uint256 constant privateKeyDeployer = 0xff22437ccbedfffafa93a9f1da2e8c19c1711052799acf3b58ae5bebb5c6bd7b;
+    address deployer = vm.addr(privateKeyDeployer);
+    address zeroAddress = Constants.getZeroAddress();
+    address constant origin = 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38;  //default address origin in foundry       
+    address constant mockAddress = 0xeB721f3E4C45a41fBdF701c8143E52665e67c76b;
+    address constant utilityTokenAddress = 0x4b02422DC46bb21D657A701D02794cD3Caeb17d0;
+    address constant hTokenAddress = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    uint32 constant validChainId = 5;
+    uint32 constant invalidChainId = 0;
+    uint16 constant expectedHolographableContractsCount = 6;
+    bytes32 contractHash = getContractType('HolographERC721');
+
+/**
+ * @notice Get the contract type identifier hash for a given contract name.
+ * @dev This function calculates the keccak256 hash of the provided contract name
+ * to generate a unique identifier for the contract type.
+ * @param contractName The name of the contract for which the contract type identifier is needed.
+ * @return bytes32 The keccak256 hash representing the contract type identifier.
+ */
+    function getContractType(string memory contractName) public pure returns (bytes32) {
+        bytes32 contractType = keccak256(abi.encodePacked(contractName));
+        return contractType;
+    }
+
+/**
```

Comment:
Seems like most of the blocks are not aligned on these comments. I also notice that prettier is failing in CI. Is there a reason for this? Maybe prettier just needs to be run to clean this up?